.gulp/paths.js

```diff
@@ -0,0 +1,16 @@
+/*global require,module*/
+(function pathConfiguration(require, module) {
```

Comment:
Is this necessary? Can you just use a normal module.exports and require as global keywords or ES6 syntax as well?